docs/config.md

```diff
@@ -366,13 +366,13 @@ soulseek:
 
 ### Timeouts
 
-Timeout options control how long the application waits for connections to connect, and how long connections can be inactive before they are disconnected.
+Timeout options control how long the application waits for connections and how long connections can be inactive before they are disconnected.
```

Comment:
```suggestion
Timeout options control how long the application waits for connections to connect, and how long connections can be inactive before they are disconnected.
```

This is a nuanced one, since a wait for the connection to connect and a wait for the connection to close/do work are different.